README.md

```diff
@@ -92,6 +106,9 @@ spec:
 * If needed, you can use [ConfigMap](http://kubernetes.io/docs/user-guide/configmap/)
 to overwrite the `config/`.
 
+## Start Standalone
+`node-problem-detector -apiserver=http://APISERVER_IP:APISERVER_PORT?inClusterConfig=false&apiVersion=v1`
```

Comment:
Add a bit more explanation here.

To run node problem detector in standalone with insecure connection to apiserver:
``` shell
node-problem-detector -apiserver=http://APISERVER_IP:APISERVER_INSECURE_PORT?inClusterConfig=false
```
And it would be better to add another example, possibly using `serviceAccount` or `kubeconfig`.